app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,40 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @foods = current_user.foods.order(created_at: :desc)
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = current_user.foods.build(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to foods_url, notice: 'Food was successfully created.' }
+      else
+        format.html { render :new, alert: 'Food could not be created' }
+      end
+    end
+  end
+
+  def destroy
+    @food.destroy
+    respond_to do |format|
+      format.html { redirect_to foods_url, notice: 'Food was successfully destroyed.' }
+    end
+  end
```

Comment:
- [ ] I can not delete food, kindly fix this
![Screenshot from 2023-03-17 23-00-56](https://user-images.githubusercontent.com/66097371/226060910-e09b6dd2-17f6-4eae-b04d-b2a46e9cdc0e.png)

Reply:
Thank you @VICTOR-CHUKWU , we will fix it!